esx_service/vmdk_ops.py

```diff
@@ -1318,10 +1325,11 @@ def disk_detach(vmdk_path, vm):
     if not device:
        # Could happen if the disk attached to a different VM - attach fails
        # and docker will insist to sending "unmount/detach" which also fails.
```

Comment:
Not this change, Docker doesn't send unmount if mount fails anymore. But this service doesn't haven't to assume such behavior. Possibly remove all mention docker in the service code.